samples/urlshortener/src/main/kotlin/com/squareup/urlshortener/UrlShortenerServiceModule.kt

```diff
@@ -0,0 +1,20 @@
+package com.squareup.urlshortener
+
+import com.google.inject.AbstractModule
+import misk.environment.Environment
+import misk.web.WebActionModule
+import misk.web.WebModule
+import misk.web.actions.DefaultActionsModule
+
+/** Binds all service dependencies including service-specific dependencies. */
+class UrlShortenerServiceModule : AbstractModule() {
```

Comment:
why this and `UrlShortenerModule`?

Reply:
my guess is to support testing? we probably want to give some guidance on how to structure modules

Reply:
Definitely. And I think we need examples like this so that our advice is credible.